### PR TITLE
Gold elements tests

### PR DESCRIPTION
**Pull request by vsheyanov**
2 or 3 test are failing (related to display:none). Those tests are outdated in original repo.

### CODE REVIEW

File: test/gold_cc_ccv_input_test.dart [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      var error = input.querySelector('paper-input-error');
+      expect(error, isNotNull);
+
+      // The error message is only displayed after manual validation.
+      expect(error.getComputedStyle().display, equals('none'),
+          reason: 'error should be display:none');
+      input.validate();
+      expect(error.getComputedStyle().display, isNot('none'),
+          reason: 'error should not be display:none');
+    }, skip: 'https://github.com/dart-lang/polymer_elements/issues/3');
```
Comment (jakemac53) [inline test/gold_cc_ccv_input_test.dart]: Can you create a new issue for this failure and then link that here, then we can close #3.

File: test/gold_cc_expiration_input_test.dart [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      var label = input.querySelector('label').id;
+
+      expect(month, isNotNull);
+      expect(year, isNotNull);
+
+      expect(month.getAttribute('aria-labelledby'), isNotNull);
+      expect(year.getAttribute('aria-labelledby'), isNotNull);
+
+      expect(
+          month.getAttribute('aria-labelledby'), equals(label + ' monthLabel'));
```
Comment (jakemac53) [inline test/gold_cc_expiration_input_test.dart]: lets change this to string interpolation (`'${label}monthLabel'`), ditto for the next one

File: test/gold_cc_input_test.dart [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      var error = input.querySelector('paper-input-error');
+      expect(error, isNotNull);
+
+      // The error message is only displayed after manual validation.
+      expect(error.getComputedStyle().display, equals('none'),
+          reason: 'error should be display:none');
+      input.validate();
+      expect(error.getComputedStyle().display, isNot('none'),
+          reason: 'error should not be display:none');
+    }, skip: 'https://github.com/dart-lang/polymer_elements/issues/3');
```
Comment (jakemac53) [inline test/gold_cc_input_test.dart]: Lets create a new issue for this one as well, or possibly one issue for all skipped gold-element tests.

File: test/gold_email_input_test.dart [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  void testEmail(address, valid) {
+    GoldEmailInput input = fixture('basic');
+    forceXIfStamp(input);
+
+    var container = input.querySelector('paper-input-container');
+    expect(container, isNotNull);
+
+    input.value = address;
+    input.jsElement.callMethod('_onInput');
+    var errorString = address + ' should be ' + (valid ? 'valid' : 'invalid');
```
Comment (jakemac53) [inline test/gold_email_input_test.dart]: lets switch this to string interpolation as well `'$address should be ${valid ? 'valid' : 'invalid'}'`

**Comment by jakemac53** [comment]
Awesome thanks! Just a few small comments, then I can go ahead and merge.

**Comment by vsheyanov** [comment]
done!

**Comment by jakemac53** [comment]
Great, thanks again!

**Pull request merged**